docs/tutorials/vertical-fl.md

```diff
@@ -35,22 +36,19 @@ python -m pip install -r ./examples/pipelines/requirements.txt
 ```
 
 ## Data provisioning
-The data format for VFL is different from regular FL. That is why each of our examples contains its own script for uploading data that are needed for a given example.
+The data format for VFL is different from regular FL. That is why each of our examples contains its own script for uploading training data.
 
-> Note: This will split the data such that each contributor owns its portion of the features and host own only the labels
+> Note: This will split the data such that each contributor owns its portion of the features and the host owns only the labels (except for the Bank Marketing Prediction example).
 
 ### CCFRAUD
-
-Please follow steps in [CCFRAUD - Add your Kaggle credentials to the workspace key vault](../real-world-examples/ccfraud.md#Add-your-Kaggle-credentials-to-the-workspace-key-vault). Afterwards, follow same steps as for **MNIST** and **please do not forget to replace `--example MNIST_VERTICAL` with `--example CCFRAUD_VERTICAL`**).
+Please follow the steps in [CCFRAUD - Run a job to download and store the dataset in each silo](../real-world-examples/ccfraud.md#run-a-job-to-download-and-store-the-dataset-in-each-silo). You will need to have your Kaggle credentials stored in a key vault, as explained in that same page linked above.
```

Comment:
What about rest of this part?

Afterwards, follow same steps as for **MNIST** and **please do not forget to replace `--example MNIST_VERTICAL` with `--example CCFRAUD_VERTICAL`**).

Reply:
good catch, the link points to instructions that have incorrect parameter value for vertical case